modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1984,9 +1996,13 @@ private void _assignOrganizationRole(
 	}
 
 	private String _buildEqualsExpressionFilterString(
-		String fieldName, String value) {
+		String fieldName, Object value) {
+
+		if (value instanceof String) {
+			return StringBundler.concat("( ", fieldName, " eq '", value, "')");
+		}
 
-		return StringBundler.concat("( ", fieldName, " eq '", value, "')");
+		return StringBundler.concat("( ", fieldName, " eq ", value, ")");
```

Comment:
@p-albuquerque 

````java
if (value instanceof String) {
  value = "'" + value + "'"
}
````

I think we have a method to surround a string with quotes.